java/code/src/com/redhat/rhn/taskomatic/task/threaded/TaskQueue.java

```diff
@@ -219,6 +221,8 @@ private synchronized boolean isTaskQueueDone() {
     }
 
     private synchronized void unsetTaskQueueDone() {
-        taskQueueDone = false;
+        synchronized (this) {
+            taskQueueDone = false;
+        }
```

Comment:
What effect does this change have to the other jobs which use TaskQueue ?

Reply:
Actually I think it's useless, I'm going to remove it